README.md

```diff
@@ -1 +1,66 @@
 # Weather-App
+
+> Weather-App, creating a weather forecast site using the weather API.
+
+![screenshot](https://i.imgur.com/6wIrIqP.png)
+
+The main goal of this project is create a weather forecast site using the weather API from the previous lesson and be able to search for a specific location and toggle displaying the data in Fahrenheit or Celsius.
+
+## Built With
+
+- HTML
+- SCSS
+- Bootstrap
+- JavaScript
+- Webpack
+- Weather API
+
+## Live Demo
+
+[Live Demo Link](https://rawcdn.githack.com/SofiBretz/Weather-App/2a52a6251baf9d99149a8c41539ee5d3bf7f4ae5/dist/index.html)
+
+## Getting Started
+
+- Clone this project on your computer.
+- Open the files on you text editor.
+- Open the .html on your browser to see the future changes.
+
+### Prerequisites
+
+- You have to get a text editor to inspect the code in your computer.
+
+### Install
+
+- Install a text editor (if you don't have already).
+- Clone the project to your computer.
+
```

Comment:
specify the commands to install and setup the project

Reply:
Thanks for this.